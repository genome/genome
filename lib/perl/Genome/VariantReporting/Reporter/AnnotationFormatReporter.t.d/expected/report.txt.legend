Headers
chromosome_name	Chromosome: an identifier from the reference genome or an angle-bracketed ID String ("<ID>") pointing to a contig in the assembly file
start	The start position of the variant. One-based
stop	The end position of the variant. One-based, inclusive
reference	The reference allele at this position
variant	The variant called at this position
type	undefined
gene_name	undefined
transcript_name	Ensembl stable ID of feature
transcript_species	undefined
transcript_source	undefined
transcript_version	undefined
strand	undefined
transcript_status	undefined
trv_type	Consequence type of this variation
c_position	The HGVS coding sequence name
amino_acid_change	The HGVS protein sequence name
ucsc_cons	undefined
domain	undefined
all_domains	undefined
deletion_substructures	undefined
transcript_error	undefined
default_gene_name	The gene symbol
gene_name_source	The source of the gene symbol
ensembl_gene_id	Ensembl stable ID of affected gene
Filters
