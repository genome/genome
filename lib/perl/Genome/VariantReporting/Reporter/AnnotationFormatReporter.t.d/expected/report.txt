chromosome_name	start	stop	reference	variant	type	gene_name	transcript_name	transcript_species	transcript_source	transcript_version	strand	transcript_status	trv_type	c_position	amino_acid_change	ucsc_cons	domain	all_domains	deletion_substructures	transcript_error	default_gene_name	gene_name_source	ensembl_gene_id
1	1	1	A	T	-	-	ENST00000452176	-	-	-	-	-	DOWNSTREAM	c.456		-	-	-	-	-	RP5-857K21.5	HGNC	ENSG00000223659
1	1	1	A	G	-	-	ENST00000452176	-	-	-	-	-	DOWNSTREAM	c.456		-	-	-	-	-	RP5-857K22.5	HGNC	ENSG00000223695
